utility_code/lib_py3/item_replacement_substitutions.py

```diff
@@ -204,18 +204,45 @@ def process(self, item_meta, item):
         if item.nbt.has_path("tag"):
             update_plain_tag(item.nbt.at_path("tag"))
 
-class UpdateShattered(SubstitutionRule):
-    """Note: To be deleted after the grave rework went live."""
-    name = "Update Shattered"
+class UpdateRenamedShulkers(SubstitutionRule):
+    """Note: To be deleted after one week!"""
+    name = "Update renamed Shulker Boxes"
 
     def process(self, item_meta, item):
-        if not item.nbt.has_path('tag.Monumenta.PlayerModified.Shattered'):
+        old_name = item_meta['name']
+        if old_name is None or not item_meta['id'].endswith('shulker_box'):
             return
-        del item.nbt.at_path('tag.Monumenta.PlayerModified').value['Shattered']
-        if not item.nbt.has_path('tag.Monumenta.PlayerModified.Infusions'):
-            item.nbt.at_path('tag.Monumenta.PlayerModified').value['Infusions'] = nbt.TagCompound({})
-        item.nbt.at_path('tag.Monumenta.PlayerModified.Infusions').value['Shattered'] =\
-            nbt.TagCompound({'Infuser': nbt.TagString("00000000-0000-0000-0000-000000000000"), 'Level': nbt.TagInt(3)})
+        new_name = None
+        new_custom_name = None
+        if re.fullmatch(r'[a-zA-Z0-9_]{3,16}', old_name):  # MC account name, or any of the selectable names at Johnny Hammer
```

Comment:
I'd suggest compiling this and using the compiled regex to speed up the update process.